README.md

```diff
@@ -12,12 +12,13 @@ To help you pick which APK to install you can find information about the version
 
 The APKs are named as follows: `cht-android-{version}-{brand}-{rendering-engine}-{instruction-set}-release.apk`
 
-| Rendering engine | Instruction set | Android version | Notes |
-|------------------|-----------------|-----------------|--|
-| `webview`        | `arm64-v8a`     | 10+             |  Preferred. Use this APK if possible. |
+| Rendering engine | Instruction set | Android version | Notes                                                       |
+|------------------|-----------------|-----------------|-------------------------------------------------------------|
+| `webview`        | `arm64-v8a`     | 10+             | Preferred. Use this APK if possible.                        |
 | `webview`        | `armeabi-v7a`   | 10+             | Built but not compatible with any devices. Ignore this APK. |
```

Comment:
Unrelated question to this PR:

Why do we build the `armeabi-v7a` APK if it can't be used? 🤔 Can we ask to remove it from the build to avoid confusion?

Reply:
I think it automatically builds every combination of every dimension, so I don't think we can stop it. We _could_ delete it when the build finishes with a custom script. It's probably not worth spending time on at this point as it will almost certainly go away when we solve [this issue](https://github.com/medic/cht-android/issues/164) because after that we expect to build one bundle instead of a series of apks.